Integration/SPM/SwiftPackageManager.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -6,8 +6,8 @@
         "repositoryURL": "https://github.com/kean/Nuke.git",
         "state": {
           "branch": null,
-          "revision": "961597d324949c7003c77d602bfc4d5519b47e81",
-          "version": "9.2.4"
+          "revision": "472a03c15f39592007d94f25abe554350bc8aa2b",
+          "version": "10.5.1"
```

Comment:
Shouldn't we discard this change? We have not have enough time to check if there's any issue that this can cause.

Reply:
@polqf I was thinking the same. We can simply discard that change on Nuno's PR as it's non-essential for the work alone.